Include/object.h

```diff
@@ -332,7 +332,8 @@ static inline Py_ssize_t Py_SIZE(PyObject *ob) {
 #  define Py_SIZE(ob) Py_SIZE(_PyObject_CAST(ob))
 #endif
 
-static inline Py_ALWAYS_INLINE int _Py_IsImmortal(PyObject *op)
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 < 0x030d0000
+inline Py_ALWAYS_INLINE int _Py_IsImmortal(PyObject *op)
```

Comment:
Judging by CI errors, you also need the linkage to be consistent here.
```suggestion
inline Py_ALWAYS_INLINE PyAPI_FUNC(int) _Py_IsImmortal(PyObject *op)
```